app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/MainActivity.java

```diff
@@ -51,201 +58,121 @@
-        descArray.add("A bebida perfeita para quem quer entrar no mundo do café. Combinação do leite cremoso, chocolate e um toque de café 100% Arábica.");
-        descArray.add("O sabor do achocolatado mais famoso do Brasil, em uma só cápsula. Contém Active-Go: um composto exclusivo com cálcio, ferro e vitaminas para o seu café da manhã ficar ainda mais completo.");
-        descArray.add("Para deixar ainda mais brasileiro e associá-lo a sua ocasião perfeita: o Café da Manhã!");
-        descArray.add("Grãos 100% Arábica com torrefação média, aroma frutado e torrado. Intenso sabor aromático, ligeiramente picante, com notas persistentes no paladar mesmo após degustação. Pode ser degustado no café da manhã ou em qualquer hora do dia.");
-        descArray.add("A exótica combinação do chá preto, leite cremoso e uma seleta combinação de especiarias: canela, gengibre cardamomo e cravo.");
-        descArray.add("Mergulhe na maravilha do chá marroquino. Uma mistura do aromático chá verde, menta intensamente saborosa e refrescante, coberto com a cremosa espuma e adoçado com perfeição.");
-
-
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(1);
-        tracosCapsula1Array.add(1);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(1);
-        tracosCapsula1Array.add(6);
-        tracosCapsula1Array.add(1);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(3);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(2);
-        tracosCapsula1Array.add(7);
-        tracosCapsula1Array.add(7);
-        tracosCapsula1Array.add(3);
-        tracosCapsula1Array.add(3);
-        tracosCapsula1Array.add(3);
-        tracosCapsula1Array.add(4);
-        tracosCapsula1Array.add(4);
-        tracosCapsula1Array.add(4);
-        tracosCapsula1Array.add(3);
-        tracosCapsula1Array.add(6);
-
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(6);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(0);
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public void updateDB() throws IOException {
```

Comment:
You should maybe call this something different, more related with what the method does.
Something in the lines of startCoffeeFetchRequest or something similar.